niworkflows/utils/spaces.py

```diff
@@ -415,7 +416,7 @@ def __contains__(self, item):
 
     def __repr__(self):
         """Representation of this object."""
-        return 'Imaging spaces: %s' % ', '.join(self._spaces)
+        return 'Imaging spaces: %s' % ', '.join([s.fullname for s in self.spaces])
```

Comment:
I agree we need to revise this, but I would go this way (not to drop specs):

```suggestion
        return 'Imaging spaces:%s' % '\n  - '.join([] + [str(s) for s in self.spaces])

Reply:
WDYT?

Reply:
sure this is cleaner